File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/playerWithCard/AllPlayerWithCardsModel.java
package com.syahputrareno975.cardbattlemodule.model.playerWithCard;

import java.io.Serializable;
import java.util.ArrayList;

public class AllPlayerWithCardsModel implements Serializable {
    public ArrayList<PlayerWithCardsModel> Players = new ArrayList<PlayerWithCardsModel>();

    public AllPlayerWithCardsModel() {
    }

    public AllPlayerWithCardsModel(ArrayList<PlayerWithCardsModel> players) {
        Players = players;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/playerWithCard/AllPlayerWithCardsModel.kt
package com.syahputrareno975.simplecardbattle.model.playerWithCard

import java.io.Serializable

class AllPlayerWithCardsModel :Serializable {
    var Players = ArrayList<PlayerWithCardsModel>()

    constructor()

    constructor(Players: ArrayList<PlayerWithCardsModel>) {
        this.Players = Players
    }


}